src/providers/drpc.ts

```diff
@@ -0,0 +1,22 @@
+import { CHAIN_IDs, MAINNET_CHAIN_IDs as _MAINNET_CHAIN_IDs, PUBLIC_NETWORKS } from "../constants";
+import { RPCTransport } from "./types";
+
+const MAINNET_CHAIN_IDs = Object.values(_MAINNET_CHAIN_IDs);
```

Comment:
I think this is unused

Reply:
Yeah that's right. My mistake. https://github.com/across-protocol/sdk/pull/706/commits/bc0213e7f71b59713540d0cc1481deb3f531ad7a